src/tiled/tileseteditor.cpp

```diff
@@ -196,7 +196,13 @@ void TilesetEditor::addDocument(Document *document)
 
     TilesetView *view = new TilesetView(mWidgetStack);
     view->setTilesetDocument(tilesetDocument);
-    view->setZoomable(new Zoomable(this));
+
+    auto zoomable = new Zoomable(this);
+    auto fileName = tilesetDocument->fileName();
+    qreal zoom = Preferences::instance()->TilesetScale(fileName);
```

Comment:
> If to store two different zooms, you need to add path parameter to Preferences::tileSetScale and call it from TilesetDock or TilesetEditor, TileAnimationEditor

Let's keep the behavior more like it is already, and only add the per-tileset zoom level to the TilesetEditor. For the TileAnimationEditor and the TilesetDock, and actually also the TileCollisionEditor, it would already be a nice improvement if they'd simply store and restore their current zoom level.